### PR TITLE
Fixed the old method of Importing Icons 

### LINKED ISSUE #7771: Fixing Icons
### Description

The PR https://github.com/EddieHubCommunity/LinkFree/pull/7738/files is a great example of how we should import icons, in the `manage-forms` branch PR https://github.com/EddieHubCommunity/LinkFree/pull/6998/files the icons are imported in the old (wrong) way (used the documentation example but tree shaking not working)

### PR DESCRIPTION
**Pull request by rohan-kulkarni-25**
## Fixes Issue 

fixes #7771 

## Changes proposed
Have updated the new import method wherever it was possible. Few imports from 'fa' were not able to transfer to the new way so left them as it is. 

## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/7778"></a>

### CODE REVIEW

File: components/GitHubAccelerator.js
Comment (yoyoekta) [inline components/GitHubAccelerator.js:1]: Isn't it the same?

File: components/account/manage/navigation.js
Comment (yoyoekta) [inline components/account/manage/navigation.js:1]: Nothing has been changed over here. If possible can you exclude this file from the PR?

**Review by yoyoekta — APPROVED** [review]

File: components/GitHubAccelerator.js [hunk 1]
```diff
@@ -1,5 +1,5 @@
 import Image from "next/image";
-import { FaGithub, FaMoneyBillAlt } from "react-icons/fa";
```
Comment (eddiejaoude) [inline components/GitHubAccelerator.js:2]: Not sure I understand this change?

File: components/account/manage/navigation.js [hunk 2]
```diff
@@ -9,6 +9,7 @@ import {
   MdOutlineBadge,
 } from "react-icons/md";
 
+
```
Comment (eddiejaoude) [inline components/account/manage/navigation.js:12]: Extra blank line not needed

```suggestion
```

File: components/account/manage/edit.js [hunk 3]
```diff
@@ -1,4 +1,4 @@
-import { PencilIcon } from "@heroicons/react/24/outline";
+import PencilIcon from "@heroicons/react/24/outline/PencilIcon";
```
Comment (eddiejaoude) [inline components/account/manage/edit.js:1]: Thanks 👍

File: components/GitHubAccelerator.js
Comment (eddiejaoude) [reply components/GitHubAccelerator.js:1]: Good spot, I will revert 👍

File: components/GitHubAccelerator.js [hunk 4]
```diff
@@ -1,5 +1,5 @@
...
+import { FaMoneyBillAlt, FaGithub } from "react-icons/fa";
```
Comment (eddiejaoude) [inline components/GitHubAccelerator.js:2]: Removing unnecessary change

```suggestion
import { FaGithub, FaMoneyBillAlt } from "react-icons/fa";
```

**Pull request merged**